rust/psibase_macros/src/fracpack_macro.rs

```diff
@@ -312,6 +321,24 @@ fn process_struct(
             })
             .fold(quote! {false}, |acc, new| quote! {#acc || #new})
     };
+
+    let fixed_size = fields
+        .iter()
+        .map(|field| {
+            let ty = &field.ty;
+            quote! {<#ty as #fracpack_mod::Pack>::FIXED_SIZE}
+        })
+        .fold(quote! {0}, |acc, new| quote! {#acc + #new});
+
+    let heap_size = fields
```

Comment:
The name `heap_size` is somewhat confusing. We've been using the term "heap" to refer to the region after the fixed size fields.

Reply:
I agree. This name was introduced by Todd and I just kept using it... not sure if in the v0 implementation it was in fact just the heap size and as it evolves it does not make sense anymore...

What name would you like to suggest here? Notice that it is only used for the packing fn and it does this assertion right below:

```
let heap = #heap_size;
assert!(heap as u16 as u32 == heap); // TODO: return error
```

Reply:
`heap` makes sense. `heap_size` doesn't. It's the size of the fixed data region, which is equal to the offset of the start of the heap.